### PR TITLE
Support dev builds in install_couchbase_server.py

### PR DESCRIPTION
**Pull request by sethrosetter**
Add support to target dev builds of Couchbase Server

### CODE REVIEW

File: libraries/provision/install_couchbase_server.py [hunk 1]
```diff
@@ -9,62 +9,69 @@ class CouchbaseServerConfig:
...
-            print "Server package url not found. Make sure to specify a version / build."
-            raise ValueError("Invalid server url")
 
-    @property
-    def version(self):
-        return self._version
+            # Get dev server package from latestbuilds
+            if self.version.startswith("4.1"):
+                # http://172.23.120.24/builds/latestbuilds/couchbase-server/sherlock/5914/couchbase-server-enterprise-4.1.1-5914-centos7.x86_64.rpm
+                base_url = "http://172.23.120.24/builds/latestbuilds/couchbase-server/sherlock/{}".format(self.build)
```
Comment (tleyden) [inline libraries/provision/install_couchbase_server.py]: is there a dns name associated w/ 172.23.120.24?

Comment (tleyden) [inline libraries/provision/install_couchbase_server.py]: also, will this work on aws?

Comment (sethrosetter) [inline libraries/provision/install_couchbase_server.py]: > is there a dns name associated w/ 172.23.120.24?

I dont believe so. 



> also, will this work on aws?

Released versions (i.e. 4.1.0) will still look to s3 but dev builds (4.5.0-XXX, any version with a build number) of server will not work in AWS due to being behind VPN. This should not block perf or most mobile testing purposes for the time being as long as you depend on a released version.

Comment (tleyden) [inline libraries/provision/install_couchbase_server.py]: Can you update to cbnas01.sc.couchbase.com?

**Pull request merged**